File: javascript/fibonacci.js
function fibonacci(n) {
  const fib = [0, 1];
  for (i = 2; i < n; i++) {
    fib[i] = fib[i - 1] + fib[i - 2];
  }

  return fib;
}

console.log(fibonacci(6));

// function fibonacciWhile(n) {
//     let fi

// }

File: rust/src/bin/fibonacci.rs
fn fibonacci(n:usize) -> Vec<u64> {
    let mut fib = vec![0, 1];
    for i in 2..n {
        fib.push(fib[i-1] + fib[i-2]);
    }

    fib
}

fn main() {
    println!("{:?}", fibonacci(6))
}